cts.schema.json

```diff
@@ -45,12 +48,31 @@
           "required": [
             "document",
             "result"
-          ]
+          ],
+          "properties": {
+            "invalid_selector": false,
+            "results": false
+          }
+        },
+        {
+          "required": [
+            "document",
+            "results"
+          ],
+          "properties": {
+            "invalid_selector": false,
+            "result": false
+          }
         },
         {
           "required": [
             "invalid_selector"
-          ]
+          ],
+          "properties": {
+            "document": false,
```

Comment:
@gregsdennis I took the liberty of tightening this up, even though it's not needed for the current PR.